src/main/cpp/fielddata/FieldElementCalculator.h

```diff
@@ -104,21 +104,21 @@ class FieldElementCalculator
     frc::Transform3d m_calcCageLeft = frc::Transform3d(
         frc::Translation3d(
             units::length::inch_t(-18.0),
-            units::length::inch_t(-42.0),
+            units::length::inch_t(-43.0),
```

Comment:
Again constants might be nice to help document these.  Please make another story to modify these.

constexpr units::length::inch_t centerOffsetFromTag = units::length::inch_t(-1.0);
constexpr units::length::inch_t distanceBetweenCages = units::length::inch_t(42.0);

Then the center can use centerOffsetFromTag and the left and right can use that + and -  the distanceBetweenCages